plugins/README.md

```diff
@@ -11,16 +11,16 @@ tag on [npm](https://www.npmjs.com/search?q=%40blockly).
 
 ### Plugins
 
-- [``@blockly/plugin-workspace-search``](plugins/workspace-search/): A plugin that adds workspace search support.
+- [``@blockly/plugin-workspace-search``](blockly-workspace-search/): A plugin that adds workspace search support.
```

Comment:
Do we want to addd typed modal and the mutator package to this list?

Reply:
Yes, I'll add to my TODO list of readme updates.